Source/OCMock/OCPartialMockObject.m

```diff
@@ -22,6 +22,31 @@
 #import "OCMFunctions.h"
 #import "OCMInvocationStub.h"
 
+/// Returns the class which should be used for mocking the object.
+Class OCPartialMockObjectClassForObject(NSObject *object);
+
+Class OCPartialMockObjectClassForObject(NSObject *object)
+{
+    /*
+     Core Data dynamically subclasses managed oject subclasses at runtime in order to implement
+     modeled setters and getters for attributes and relationships.
+     
+     If we are mocking a managed object, ensure we create a mock subclass from Core Data's private subclass, not the exposed class.
+     */
+    
+    static Class managedObjectClass = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        managedObjectClass = NSClassFromString(@"NSManagedObject");
```

Comment:
is there a reason why object_getClass() should not be used always?  If we are mocking something, seems like we should be invoking methods on the actual class no matter what.